src/main/java/org/fundacionjala/coding/ovidio_spinwords/SpinWord.java

```diff
@@ -0,0 +1,48 @@
+package org.fundacionjala.coding.ovidio_spinwords;
+import java.util.StringJoiner;
+
+
+
+/**
+ * Created by OvidioMiranda on 5/19/2017.
+ */
+public class SpinWord {
+
+    private String sentence;
+
+
+    /**
+     * Constructor for objects of SpinWord.
+     */
+    public SpinWord() {
+        this("");
+    }
+
+    /**
+     * Constructor for objects of SpinWord with a parameter.
+     *
+     * @param sentence Is a word.
+     */
+    public SpinWord(String sentence) {
+        this.sentence = sentence;
+    }
+
+
+    /**
+     * Function that takes in a string and returns the same string
+     * but with all five or more letter words reversed.
+     *
+     * @return returns the same string but with all five or more letter words reversed .
+     */
+    public String spinWords() {
+        final Integer maxLetters = 5;
```

Comment:
please use int instead of using Integer because you are not handling null values. :)